spec/cc/config/merge_spec.rb

```diff
@@ -60,6 +60,30 @@
           expect(engine.config).to eq(foo: "baz", meow: { "yo": "sup", "foo": "bar" })
         end
       end
+
+      it "defaults to cronopio channel for duplication" do
+        config1 = CC::Config.new(
+          engines: [
+            CC::Config::Engine.new("duplication", enabled: true, config: { languages: "java" }),
+          ].to_set,
+        )
+
+        config2 = CC::Config.new(
+          engines: [
+            CC::Config::Engine.new("duplication", enabled: true, config: { languages: "ruby" }),
+          ].to_set,
+        )
+
+        merged_config = described_class.new(config1, config2).run
+
+        expect(merged_config.engines.count).to eq(1)
+
+        merged_config.engines.to_a.first.tap do |engine|
+          expect(engine.name).to eq("duplication")
+          expect(engine.config).to eq(languages: "ruby")
+          expect(engine.channel).to eq(CC::Config::Engine::DUPLICATION_CHANNEL)
+        end
+      end
```

Comment:
I think it could be worth covering just the changed behavior: that for duplication it uses cronopio, and for something else, it uses "stable". That second part seems not to be explicitly covered (not that I think it won't work). And the merging behavior probably doesn't need to be covered in this spec, in the interest of keeping specs focused